parsec/core/win_registry.py

```diff
@@ -187,7 +187,7 @@ def _get_drive_icon_path(device: LocalDevice) -> Traversable:
 @contextmanager
 def parsec_drive_icon_context(letter: str, device: LocalDevice) -> Iterator[None]:
     # Winreg is not available for some reasons
-    if sys.platform != "win32" or not try_winreg():
+    if sys.platform != "win32" or not try_winreg() or not letter:
```

Comment:
The drive icon context should not be use in single mountpoint mode (since no drive letter is used). The `parsec_drive_icon_context` is called with `letter=""`.

An alternative solution could be to use [contextlib.nullcontext](https://docs.python.org/3.7/library/contextlib.html#contextlib.nullcontext) in the caller: 
```python
if single_mountpoint_enabled:
    parsec_drive_icon_context = contextlib.nullcontext(drive_letter, device=user_fs.device)

with parsec_drive_icon_context(drive_letter, device=user_fs.device):
    ...
```